<launch>

    <arg name="image_src" default="/image_raw"/>
    <arg name="camera_info_src" default="/camera_info"/>
    <arg name="image_rectifier_out" default="image_rectified">

    <node pkg="image_rectifier" type="image_rectifier_node" name="image_rectifier" output="screen">
        <param name="image_src" value="${arg image_src}"/>
        <param name="camera_info_src" value="${arg camera_info_src}"/>
        <param name="image_rectifier_out" value="${arg image_rectifier_out}"/>
    </node>
    
</launch>